src/Google.Api.CommonProtos/Type/MoneyPartial.cs

```diff
@@ -12,14 +12,17 @@ namespace Google.Type
     public partial class Money
     {
         /// <summary>
-        /// The amount of money in <see cref="decimal"/> format.
+        /// The amount of money in <see cref="decimal"/> format. This is an abstraction of the Units and Nanos properties.

```

Comment:
Typo: abastraction => abstraction. (I can fix it later if you'd prefer.) Otherwise, feel free to commit --amend and force push.

Reply:
Should be fixed in the latest commit.  Thanks Jon!